api/tests/integration/edge_api/identities/test_edge_identity_featurestates_view.py

```diff
@@ -392,6 +494,65 @@ def test_edge_identities_update_featurestate(
     assert args[0] == expected_identity_document
 
 
+def test_edge_identities_with_identifier_update_featurestate(
+    admin_client,
+    environment,
+    environment_api_key,
+    identity_document,
+    dynamo_wrapper_mock,
+    feature,
+):
+    # Given
+    dynamo_wrapper_mock.get_item.return_value = identity_document
+    identifier = identity_document["identifier"]
+    featurestate_uuid = identity_document["identity_features"][0]["featurestate_uuid"]
```

Comment:
Again, I'm a bit concerned about where people will obtain the featurestate_uuid from.